File: 01.CSharp-Programming-Basics/04.Simple-Operations/HomeWork/SimpleOperations/PetShop/PetShop.cs
using System;

namespace PetShop
{
    class PetShop
    {
        static void Main(string[] args)
        {
            int dogCount = int.Parse(Console.ReadLine());
            int otherPets = int.Parse(Console.ReadLine());

            double sum = (dogCount * 2.50) + (otherPets * 4);

            Console.WriteLine($"{sum:f2} lv.");
        }
    }
}

File: 02.JS-Programming-Basics/01.First-Steps-In-Coding/Lab/PetShop.js
function petShop(dogCount, anotherAnimals){

    let totalSum = 0;
    totalSum += Number(dogCount) * 2.50;
    totalSum += Number(anotherAnimals) * 4;

    console.log(`${totalSum} lv.`)
}

petShop("5","4");
petShop("13","9");